tests/shared/src/test/scala/scala/meta/tests/parsers/TermSuite.scala

```diff
@@ -112,6 +112,18 @@ class TermSuite extends ParseSuite {
     assertTerm("f(x = xs: _*)") {
       Term.Apply(Term.Name("f"), List(Assign(Term.Name("x"), Repeated(Term.Name("xs")))))
     }
+    val Term.Apply.Until_4_6_0(
```

Comment:
I am wondering what would be the use case for this one, as source compiling against 4.6.0 would rather use the existing/previous extractor with warnings or the new `After` one.

Reply:
i thought about it, too. added `Until_` "for completeness", as an alias for the earlier `After_` but can remove it if it's confusing or unnecessary.